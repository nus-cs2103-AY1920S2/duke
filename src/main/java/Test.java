public class Test {
    public static void main(String[] args) {
        String strArr = "done 1";
        for (String s : strArr.split(" ")) {
            System.out.println(s);
        }
        System.out.println(strArr.split(" ")[0].equals("done"));
    }
}
