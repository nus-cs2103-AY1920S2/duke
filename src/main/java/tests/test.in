list
todo read book
deadline return book /by June 6th
event project meeting /at Aug 6th 2-4pm
todo join sports club
list
done 1
done 4
todo borrow book
list
deadline return book /by Sunday
event project meeting /at Mon 2-4pm
deadline do homework /by no idea :-p
blah
todo
deadline
event
deadline test /by
deadline test /
deadline test
event test /at
event test /
event test
delete
delete 20
delete 2
done
done 20
done 2
bye