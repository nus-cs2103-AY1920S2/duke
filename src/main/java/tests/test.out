Hello from
 ____        _        
|  _ \ _   _| | _____ 
| | | | | | | |/ / _ \
| |_| | |_| |   <  __/
|____/ \__,_|_|\_\___|

______________________________________________________________________________________
Hello! I'm Duke
What can I do for you?
______________________________________________________________________________________
______________________________________________________________________________________
You currently have no tasks in your list
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[T][N] read book
Now you have 1 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[D][N] return book  (by: Feb 21 2020 4PM)
Now you have 2 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[E][N] project meeting  (at: Apr 13 2020 6PM)
Now you have 3 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[T][N] join sports club
Now you have 4 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Here are the tasks in your list:
1. [T][N] read book
2. [D][N] return book  (by: Feb 21 2020 4PM)
3. [E][N] project meeting  (at: Apr 13 2020 6PM)
4. [T][N] join sports club
______________________________________________________________________________________
______________________________________________________________________________________
Nice! I've marked this task as done: 
[T][Y] read book
______________________________________________________________________________________
______________________________________________________________________________________
Nice! I've marked this task as done: 
[T][Y] join sports club
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[T][N] borrow book
Now you have 5 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Here are the tasks in your list:
1. [T][Y] read book
2. [D][N] return book  (by: Feb 21 2020 4PM)
3. [E][N] project meeting  (at: Apr 13 2020 6PM)
4. [T][Y] join sports club
5. [T][N] borrow book
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[D][N] return book  (by: Jun 5 2020 2PM)
Now you have 6 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[E][N] project meeting  (at: Aug 17 2020 10AM)
Now you have 7 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
Got it. I've added this task:
[D][N] do homework  (by: Oct 29 2020 12PM)
Now you have 8 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! I'm sorry, but I don't know what that means :-(
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The description of a todo cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The description of a deadline cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The description of a event cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The time of a deadline cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The time of a deadline cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The time of a deadline cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The time of a event cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The time of a event cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The time of a event cannot be empty.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The task to be marked as delete must be specified
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! Task 20 does not exist
______________________________________________________________________________________
______________________________________________________________________________________
Noted. I've removed this task:
[D][N] return book  (by: Feb 21 2020 4PM)
Now you have 7 tasks in the list.
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! The task to be marked as done must be specified
______________________________________________________________________________________
______________________________________________________________________________________
☹ OOPS!!! Task 20 does not exist
______________________________________________________________________________________
______________________________________________________________________________________
Nice! I've marked this task as done: 
[E][Y] project meeting  (at: Apr 13 2020 6PM)
______________________________________________________________________________________
______________________________________________________________________________________
Bye. Hope to see you again soon!
______________________________________________________________________________________
